scripts/openethereum.service

```diff
@@ -13,8 +13,10 @@ ExecStart=/usr/bin/openethereum --config /etc/openethereum/config.toml
 Restart=on-failure
 
 # Specifies which signal to use when killing a service. Defaults to SIGTERM.
-# SIGHUP gives openethereum time to exit cleanly before SIGKILL (default 90s)
-KillSignal=SIGHUP
+# SIGTERM gives openethereum time to exit cleanly (default 90s) before sending SIGKILL
+KillSignal=SIGTERM
+TimeoutStopSec=90
```

Comment:
We have a registered handler for SIGTERM and I agree that it should be there. it is ctrlc lib: https://github.com/Detegr/rust-ctrlc
Looking at https://www.freedesktop.org/software/systemd/man/systemd.kill.html#, option `SendSIGKILL=yes` is default and could be removed. 
And can you put timeoutStopSec to something like 300 just to be sure.

Reply:
Done